internal/mode/static/config/config.go

```diff
@@ -81,3 +83,13 @@ type LeaderElection struct {
 	// Enabled indicates whether leader election is enabled.
 	Enabled bool
 }
+
+// FlagKeyValues contains the parsed NGF flag keys and values.
+// Flag Key and Value are paired based off of index in slice.
+type FlagKeyValues struct {
```

Comment:
what if we simplify the name to Flags ?

And also simplify fields:
- Keys // Perhaps Names better though?
- Values